src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFinderScoring.cpp

```diff
@@ -72,10 +72,13 @@ namespace OpenMS
     defaults_.setValidStrings("write_convex_hull", {"true","false"});
     defaults_.setValue("spectrum_addition_method", "simple", "For spectrum addition, either use simple concatenation or use peak resampling", {"advanced"});
     defaults_.setValidStrings("spectrum_addition_method", {"simple", "resample"});
-    defaults_.setValue("add_up_spectra", 1, "Add up spectra around the peak apex (needs to be a non-even integer)", {"advanced"});
-    defaults_.setMinInt("add_up_spectra", 1);
+    defaults_.setValue("add_up_spectra", 1, "Add up spectra around the peak apex (needs to be a non-even integer). If set to -1, it will automatically select the number of spectra to add up around the peak apex.", {"advanced"});
+    defaults_.setMinInt("add_up_spectra", -1);
     defaults_.setValue("spacing_for_spectra_resampling", 0.005, "If spectra are to be added, use this spacing to add them up", {"advanced"});
     defaults_.setMinFloat("spacing_for_spectra_resampling", 0.0);
+    defaults_.setValue("use_percent_peak_width", 0.15, "If spectra are to be added based on the peak width of peak, constrict number of spectra to be added based on N percent of number of points of peak width.", {"advanced"});
```

Comment:
should we call it percent if it is a fraction?

Reply:
Good point, could probably change it to `fraction_of_peak_width`?

Reply:
intend could be clearer if we use something like "merge_spectra_by_peak_width_fraction"